src/callout.tsx

```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
                 {icon && <div className={classes.icon}>{icon}</div>}
```

Comment:
perhaps you could replace this div with a `<Box paddingRight={16}>` and remove the icon class